### ISSUE TITLE
Enhancement proposal: Add link to full documentation

### ISSUE THREAD

**Issue by stlehmann** [kiteco/issue-tracker#207]
First, thank you so much for this great tool. I'm using Kite as a PyCharm plugin for quite a while now and wouldn't want to miss it. 

One thing I noticed is that I often find myself using the Kite documentation for an object under the cursor. This is a very convenient feature and spares me the time and hassle to google. However, the Kite documentation on Python classes is limited and often I need some more information about the usage of a class or some parameters that need to be passed to a method. In this case I end up googling again for the class and open the official Python documentation. That's why I sometimes tend to skip using Kite documentation at all because I know in 50% of the cases I still need to look in the web for further documentation.

It would be such a timesaver if there was a link in the Kite documentation that leads me directly to the full Python documentation (or even documentation of third-party packages). This way it would always make sense to use Kite as a first option for documenation because from there I can directly get to full documentation of a package with a single click... no googling anymore :)

I hope you appreciate the feedback.
Kind Regards

**Comment by cogsci2** [comment]
Yes - with Python, it would be helpful to append the entire docstring or link to the original documentation.  I understand that linking out is a non-trivial ask because the docs are indexed from local source and documentation on the web is  a moving target.  Instead, it might be possible to link to a search.

But the easiest solution is to just supply the whole docstring, either appended to the copilot information or as a "link" to a new page.
